### PR TITLE
BookStore: Added Styling

### PR DESCRIPTION
**Pull request by Mike111222**
I have done the following changes and I want to implement these:

- Add missing JSX elements to the project (without adding extra functionality)
- Final design matches Zeplin's design (this includes all elements + the same styling)
- Updated packages to the latest version of axios.
- Checked the web App in browser.
- Checked linters errors

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -2,16 +2,29 @@ import React from 'react';
...
+        </button>
+        <div className="collapse navbar-collapse" id="navbarText">
+          <ul className="navLinks navbar-nav me-auto mb-2 mb-lg-0">
+            <li className="nav-item me-5">
+              <Link to="/">Books</Link>
+            </li>
+            <li className="nav-item">
+              <Link to="categories">Categories</Link>
+            </li>
+          </ul>
```
Comment (AdnanOlarmmi) [inline src/components/Navigation.js]: - [x] Kindly add the indication for the current navigation, as it is in the design, you could do this using states.

- Your design
image

- Zeplin design
image

File: src/components/Navigation.js [hunk 2]
```diff
@@ -2,16 +2,29 @@ import React from 'react';
...
+      <div className="container-fluid">
+        <h2 className="navTitle navbar-brand">Bookstore CMS</h2>
+        <button className="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span className="navbar-toggler-icon" />
+        </button>
+        <div className="collapse navbar-collapse" id="navbarText">
+          <ul className="navLinks navbar-nav me-auto mb-2 mb-lg-0">
+            <li className="nav-item me-5">
+              <Link to="/">Books</Link>
+            </li>
```
Comment (AdnanOlarmmi) [inline src/components/Navigation.js]: - [x] Kindly fix the bottom-border/box-shadow beneath the header section, to have a 💯 pixel perfect design.

- Your design
image

- Zeplin design
image

So, the background color of the header is also different from that of the container.

File: src/components/Navigation.js [hunk 3]
```diff
@@ -2,16 +2,29 @@ import React from 'react';
...
+            <li className="nav-item">
+              <Link to="categories">Categories</Link>
+            </li>
+          </ul>
+          <span className="navbar-text">
+            <img className="d-none d-md-none d-lg-block" src="https://img.icons8.com/external-anggara-outline-color-anggara-putra/32/000000/external-avatar-interface-anggara-outline-color-anggara-putra.png" alt="profile" />
+          </span>
+        </div>
```
Comment (AdnanOlarmmi) [inline src/components/Navigation.js]: - [x] Kindly capitalize the following to match the design exactly 😃.
image
image

File: src/components/AddBook.js [hunk 4]
```diff
@@ -22,8 +30,8 @@ const AddBook = () => {
   };
 
   return (
-    <section>
-      <h2>Add New Book</h2>
+    <section className="addBook">
+      <h2>ADD NEW BOOK</h2>
```
Comment (AdnanOlarmmi) [inline src/components/AddBook.js]: - [ ] Kindly correct the font-weight and color of this text to match the design 🤌 

- Your design
image

- Zeplin design
image

File: src/components/AddBook.js [hunk 5]
```diff
@@ -40,6 +48,19 @@ const AddBook = () => {
...
+          <option value="Mystery">Mystery</option>
+          <option value="Fantasy">Fantasy</option>
+          <option value="Historical">Historical</option>
+          <option value="Horror">Horror</option>
+          <option value="Romance">Romance</option>
+          <option value="Science">Science</option>
+        </select>
 
         <button type="submit" onClick={handleSubmit}>Add Book</button>
       </form>
```
Comment (AdnanOlarmmi) [inline src/components/AddBook.js:66]: - [ ] Kindly correct the border styling of the input field, to match the design.

- Your design
image

- Zeplin design
image

And likewise the font family of the submit button is RobotoSlab
image

File: src/components/BookCard.js [hunk 6]
```diff
@@ -1,38 +1,56 @@
...
+                  <div className="fill" />
+                </div>
+                <div className="inside-circle" />
+              </div>
+            </div>
+          </div>
+        </div>
+        <div className="percentages">
+          <h2>33%</h2>
+          <p className="status">Completed</p>
```
Comment (AdnanOlarmmi) [inline src/components/BookCard.js:49]: - [ ] [OPTIONAL] For cleaner code benefit, the progress bar could be another component 😃

File: src/components/BookCard.js [hunk 7]
```diff
@@ -1,38 +1,56 @@
...
-          <li><button type="button">Comments</button></li>
-          <li><button type="button" onClick={() => handleRemoveBook(book.itemId)}>Remove</button></li>
-          <li><button type="button">Edit</button></li>
+          <li><button className="bookBtn" type="button">Comments</button></li>
+          <li><button className="bookBtn" type="button" onClick={() => dispatch(deleteBook(book.item_id))}>Remove</button></li>
+          <li><button className="bookBtn" type="button">Edit</button></li>
         </ul>
       </div>
-      <div className="bookChart">
-        <div>Chart</div>
```
Comment (AdnanOlarmmi) [inline src/components/BookCard.js]: - [x] Kindly fix the spacing here, thanks 
image

- Zeplin design
image

And likewise, add the separator (|) between comment remove and edit

File: src/App.css
Comment (AdnanOlarmmi) [inline src/App.css:1]: - [ ] Kindly maintain consistent spacing by making the Add form have the same horizontal spacing as that before it, this is called hierarchy in design, and it's a very crucial UI principle.

- Your design
image

- Zeplin design
image

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @Mike111222, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- Nice design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navigation.js (hunk 3 above)
Comment (Mike111222) [reply src/components/Navigation.js]: I have done the changes you mentioned. Thanksa lot for your kind feedback.

File: src/components/AddBook.js (hunk 4 above)
Comment (Mike111222) [reply src/components/AddBook.js]: I have done the changes you suggested. Thanks for the feedback.

File: src/components/AddBook.js (hunk 5 above)
Comment (Mike111222) [reply src/components/AddBook.js:66]: I have implemented the said changes as per your suggestions. Thanks for the feedback.

File: src/App.css
Comment (Mike111222) [reply src/App.css:1]: I have done the changes you mentioned. Thanks for the feedback.

File: src/components/BookCard.js (hunk 7 above)
Comment (Mike111222) [reply src/components/BookCard.js]: I have done the changes as you mentioned sir. Thanks and well noted.

File: src/components/Navigation.js (hunk 2 above)
Comment (Mike111222) [reply src/components/Navigation.js]: I have implemented the changes according to your feedback. Well noted with thanks.

File: src/components/Navigation.js (hunk 1 above)
Comment (Mike111222) [reply src/components/Navigation.js]: I have done the changes you mentioned. Well noted with thanks

**Review by shella12 — APPROVED** [review]
Hi @Mike111222, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Clean UI design ✔️ 
- Implemented all required changes 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**